DAQ_System/Source/main.cpp

```diff
@@ -1,10 +1,82 @@
 #include "mbed.h"
+#include "LSM303DLHC.h"
+#include <iostream>
+
+static float _lsm303Accel_MG_LSB     = 0.001F;
+const int sampleCount = 100;
```

Comment:
Style guide's requirements for [naming constants](https://google.github.io/styleguide/cppguide.html#Constant_Names)